frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -297,7 +297,7 @@ export const getNotUpgradeableResources = (resources) =>
   resources.filter((resource) => getConditionUpgradeableFalse(resource));
 
 export enum NodeTypes {
-  master = 'master',
```

Comment:
This is problematic for https://github.com/openshift/console/blob/f4247fbf08785207f9ad57929f823167eb071c8c/frontend/public/module/k8s/cluster-settings.ts#L313